mariadb/rootfs/etc/s6-overlay/s6-rc.d/mariadb-core/run

```diff
@@ -4,7 +4,19 @@
 # Start MariaDB service
 # ==============================================================================
 
+# Initialize an empty string to hold the concatenated parameters
+param_string=""
```

Comment:
Maybe this should be an array? I wonder if your approach of simply concatenating parameters would work with an argument which contains spaces.

Reply:
The conversion to the required format is done in line 15

Reply:
In a way the solution is neat, but also only works because mariadb accepts underlines as well as dashes as argument.

Thinking a bit more about this, I think it would be better if we follow how things are done in other add-ons, like the Matter add-on: Let's have an "extra arguments" configuration, which users then can fill with whatever they feel appropriate.

See for instance: <https://github.com/home-assistant/addons/blob/master/matter_server/config.yaml#L37-L38>